elixir/word-count/word_count_test.exs

```diff
@@ -0,0 +1,62 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("word_count.exs", __DIR__)
+end
+
+ExUnit.start()
+ExUnit.configure(exclude: :pending, trace: true)
+
+defmodule WordsTest do
+  use ExUnit.Case
+
+  test "count one word" do
+    assert Words.count("word") == %{"word" => 1}
+  end
+
+  # @tag :pending
```

Comment:
Why not 🔥 those lines instead of commenting? 😆

Reply:
It is the exercism recommendation 😆 
Check [here](https://github.com/elainenaomi/exercism-tracks/pull/5/files#diff-e3197bc8c944231bc1a66ef82419edc3R30)

Reply:
I think I never followed those hahaha